.travis.yml

```diff
@@ -6,8 +6,8 @@ sudo: false
 perl6:
   - latest
 install:
-  - rakudobrew build-panda
-  - panda installdeps .
+  - rakudobrew build-zef
+  - zef --debug install .
```

Comment:
This will install it instead of installing its dependencies making test results unreliable (i.e. not a clean environment). I think you meant `zef --depsonly install .`.